index.d.ts

```diff
@@ -48,6 +48,11 @@ declare namespace ansiStyles {
 		*/
 		readonly underline: CSPair;
 
+		/**
+		Make text overline. Supported on VTE-based terminals, the GNOME terminal, mintty, and Git Bash.
```

Comment:
The readme and index.d.ts should be in sync.

Reply:
bcf6fe8